lib/qs/error_handler.rb

```diff
@@ -15,11 +23,10 @@ def initialize(exception, daemon_data, job = nil)
     # exception will be passed to subsequent error procs. This is designed to
     # avoid "hidden" errors, this way the daemon will log based on the last
     # exception that occurred.
-
     def run
       @error_procs.each do |error_proc|
         begin
-          error_proc.call(@exception, @daemon_data, @job)
+          error_proc.call(@exception, @context)
```

Comment:
@jcredding this is cool - this is all error handling should be about - the exception and any context to it.  good stuff.